src/tests/unit_x509.cpp

```diff
@@ -518,6 +518,30 @@ Test::Result test_crl_dn_name() {
    return result;
 }
 
+Test::Result test_crl_without_next_update() {
```

Comment:
I'm not sure this test is doing much vs `Path_Validation_With_Immortal_CRL` especially if as I suggested in the other comment you always parse the CRL. You should just add the 3 confirm statements to that test.

Reply:
Yep, agree. That was created last week before I got a reasonably sized CRL and root CA from a colleague for testing. I'll remove it.